erica_snider/assignments/Portfolio/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+
+    <header>
+      <div id="name"><h1>Jayne Doe</h1></div>
+      <div id="tagline"><h3>Master of shadows and the Internet!</h3></div>
```

Comment:
Since these divs only have one element each, it would be better to avoid using them and just apply the id attribute directly to the h1 and h3 tags